packages/adapter-cloudflare/index.js

```diff
@@ -90,7 +90,9 @@ export default function (options = {}) {
 					format: 'esm',
 					bundle: true,
 					loader: {
-						'.wasm': 'copy'
+						'.wasm': 'copy',
+						'.woff': 'copy',
```

Comment:
is there also a `.woff2` extension we need to add here? I just did a quick search for woff and thought I saw something about that

Reply:
Yes, I should add `.woff2` as well.